vochain/indexer/queries/vote_reference.sql

```diff
@@ -1,10 +1,10 @@
 -- name: CreateVoteReference :execresult
-INSERT INTO vote_references (
+REPLACE INTO vote_references (
```

Comment:
why replace? why would we ever want to replace an existing vote?

Reply:
Yes, that's the objective of this PR. We want to allow voters to send multiple time its vote (if the election is configured to allow so).